clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -1134,189 +1134,198 @@ describe("Fides-js TCF", () => {
     });
   });
 
-  describe("second layer embedded", () => {
-    it("automatically renders the second layer and can render tabs", () => {
-      cy.getCookie(CONSENT_COOKIE_NAME).should("not.exist");
-      cy.fixture("consent/experience_tcf.json").then((experience) => {
-        stubConfig({
-          options: {
-            isOverlayEnabled: true,
-            tcfEnabled: true,
-            fidesEmbed: true,
-          },
-          experience: experience.items[0],
-        });
-      });
-      cy.get("#fides-tab-purposes");
-      cy.get("@FidesUIShown").should("have.been.calledOnce");
-      checkDefaultExperienceRender();
-    });
-    it("automatically renders the second layer even when fides_disable_banner is true", () => {
-      cy.getCookie(CONSENT_COOKIE_NAME).should("not.exist");
-      cy.fixture("consent/experience_tcf.json").then((experience) => {
-        stubConfig({
-          options: {
-            isOverlayEnabled: true,
-            tcfEnabled: true,
-            fidesEmbed: true,
-            fidesDisableBanner: true,
-          },
-          experience: experience.items[0],
+  describe("when fides_embed is true", () => {
```

Comment:
This diff is hard to read, but I reorganized the specs to be:
* `when fides_embed is true`
  * `when fides_disable_banner is false`
    * (NEW) test the layer 1 banner is shown
  * `when fides_disable_banner is true`
    * (all existing tests go here)